tests/main.js

```diff
@@ -83,6 +84,18 @@ const Config = require('..')
 }
 
 {
+  console.info('- It calls onSuccess hook')
+  const testConfig = new Config()
+  let hookFilePath
+  testConfig.loadFile({
+    relativePath: '.no-default-name.yaml',
+    onSuccess: filePath => { hookFilePath = filePath },
+  })
+  expect(hookFilePath, 'to equal', path.resolve('.no-default-name.yaml'))
```

Comment:
Move the check into the `onSuccess` function!?

Reply:
Needs to stay outside, otherwise the test will always succeed.